tests/common/fixtures/advanced_reboot.py

```diff
@@ -94,6 +96,10 @@ def __init__(self, request, duthosts, duthost, ptfhost, localhost, tbinfo, creds
         self.lagMemberCnt = 0
         self.vlanMaxCnt = 0
         self.hostMaxCnt = HOST_MAX_COUNT
+        if "dualtor" in self.getTestbedType():
+            DUAL_TOR_MODE = True
+            other_duthost = get_peerhost(duthosts, duthost)
+            self.other_duthost = other_duthost if other_duthost != duthost else None
```

Comment:
minor: can we rename `other_duthost` to `peer_duthost`?

Reply:
Done